app/models/standing.rb

```diff
@@ -3,18 +3,17 @@
 class Standing < ApplicationRecord
   belongs_to :team
 
-  def self.compute(team)
-    # puts "Computing standing for #{team.name}"
+  def self.compute(team) # rubocop:disable Metrics/AbcSize
```

Comment:
Nit: this whole method can be reduced to a single chained call, thus eliminating the `Metrics/AbcSize` offense:

```rb
  def self.compute(team)
    Standing
      .find_or_initialize_by(team:)
      .update(
        wins: team.wins,
        draws: team.draws,
        losses: team.losses,
        goals_pro: team.goals_pro,
        goals_against: team.goals_against,
        points: (team.wins * 3) + team.draws,
        matches: team.wins + team.draws + team.losses,
      )
  end
```

Reply:
Updated but still offends the `Metrics/AbcSize`. Code in #17

Reply:
My bad. I added a bug. After its fix, Rubocop ofence was indeed solved.  :smile: